<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
border: none;
background: black;
 }
QFrame, QToolTip{
background-color: none;
border-style: inset;
border-width: 4px;
border-color: #808080;
border-radius: 10px;
padding: 2px;
color: #FFFFFF;
text-align: right;
font-size: 50px;
 }
QLabel {
border-style: none;
color: #AA0000;
font-size: 20px;
}
QLabel#lb_is_Temp {color: white;font-size: 100px;}


QProgressBar {
}
QProgressBar::Chunk{
background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 2	, fx: 0.5, fy: 1.5, stop: 0 rgba(255,30,30,255), stop: 0.2 rgba(255,30,30,255), stop: 0.3 rgba(255,30,30,60));
}

QToolButton{
background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 2	, fx: 0.5, fy: 1.5, stop: 0 rgba(255,30,30,255), stop: 0.2 rgba(255,30,30,144), stop: 0.3 rgba(255,30,30,32));
border-style: outset;
border-width: 4px;
border-color: #808080;
border-radius: 10px;
padding: 15px;
color: #FFFFFF;
text-align: right;
font-size: 13px;
background-repeat: no-repeat;
background-position: center center;
 }
QToolButton:hover{
background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 2	, fx: 0.5, fy: 1, stop: 0 rgba(255,30,30,255), stop: 0.2 rgba(255,30,30,144), stop: 0.3 rgba(255,30,30,32));
border-style: inset; }
QToolButton:pressed{
background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 2	, fx: 0.5, fy: 1, stop: 0 rgba(255,30,30,255), stop: 0.2 rgba(255,30,30,144), stop: 0.3 rgba(255,30,30,32));
border-style: inset;}




</string>
  </property>
  <property name="iconSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>341</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lb_tg_temp">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>290</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>180</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lb_tg_durcance">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>290</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lb_tg_steam">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>290</y>
       <width>41</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lb_tg_speed">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>290</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>high</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="CircProgress" name="time" native="true">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>275</width>
       <height>275</height>
      </rect>
     </property>
    </widget>
    <widget class="CircProgress" name="temp" native="true">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>275</width>
       <height>275</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lb_tg_durcance_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>290</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>min</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lb_tg_durcance_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>290</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>ml</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lb_tg_durcance_4">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>290</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>°C</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="steamButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 2	, fx: 0.5, fy: 1, stop: 0 rgba(255,30,30,255), stop: 0.2 rgba(255,30,30,144), stop: 0.3 rgba(255,30,30,32));
border-style: inset; }
QPushButton:pressed{
background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 2	, fx: 0.5, fy: 1, stop: 0 rgba(255,30,30,255), stop: 0.2 rgba(255,30,30,144), stop: 0.3 rgba(255,30,30,32));
border-style: inset;}
</string>
    </property>
    <property name="text">
     <string>Dampf</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/steam.png</normaloff>img/steam.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>360</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 2	, fx: 0.5, fy: 1, stop: 0 rgba(255,30,30,255), stop: 0.2 rgba(255,30,30,144), stop: 0.3 rgba(255,30,30,32));
border-style: inset; }
QPushButton:pressed{
background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 2	, fx: 0.5, fy: 1, stop: 0 rgba(255,30,30,255), stop: 0.2 rgba(255,30,30,144), stop: 0.3 rgba(255,30,30,32));
border-style: inset;}
</string>
    </property>
    <property name="text">
     <string>Zurück</string>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="speedButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 2	, fx: 0.5, fy: 1, stop: 0 rgba(255,30,30,255), stop: 0.2 rgba(255,30,30,144), stop: 0.3 rgba(255,30,30,32));
border-style: inset; }
QPushButton:pressed{
background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 2	, fx: 0.5, fy: 1, stop: 0 rgba(255,30,30,255), stop: 0.2 rgba(255,30,30,144), stop: 0.3 rgba(255,30,30,32));
border-style: inset;}
</string>
    </property>
    <property name="text">
     <string>Lüfter</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/speed.png</normaloff>img/speed.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="timeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 2	, fx: 0.5, fy: 1, stop: 0 rgba(255,30,30,255), stop: 0.2 rgba(255,30,30,144), stop: 0.3 rgba(255,30,30,32));
border-style: inset; }
QPushButton:pressed{
background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 2	, fx: 0.5, fy: 1, stop: 0 rgba(255,30,30,255), stop: 0.2 rgba(255,30,30,144), stop: 0.3 rgba(255,30,30,32));
border-style: inset;}
</string>
    </property>
    <property name="text">
     <string>Dauer</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/zeit.png</normaloff>img/zeit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="doorButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>360</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 2	, fx: 0.5, fy: 1, stop: 0 rgba(255,30,30,255), stop: 0.2 rgba(255,30,30,144), stop: 0.3 rgba(255,30,30,32));
border-style: inset; }
QPushButton:pressed{
background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 2	, fx: 0.5, fy: 1, stop: 0 rgba(255,30,30,255), stop: 0.2 rgba(255,30,30,144), stop: 0.3 rgba(255,30,30,32));
border-style: inset;}
</string>
    </property>
    <property name="text">
     <string>Tür</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/door.png</normaloff>img/door.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="tempButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 2	, fx: 0.5, fy: 1, stop: 0 rgba(255,30,30,255), stop: 0.2 rgba(255,30,30,144), stop: 0.3 rgba(255,30,30,32));
border-style: inset; }
QPushButton:pressed{
background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 2	, fx: 0.5, fy: 1, stop: 0 rgba(255,30,30,255), stop: 0.2 rgba(255,30,30,144), stop: 0.3 rgba(255,30,30,32));
border-style: inset;}
</string>
    </property>
    <property name="text">
     <string>Temperatur</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/temperature.png</normaloff>img/temperature.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
  </widget>
  <action name="openDialog">
   <property name="text">
    <string>openDialog</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CircProgress</class>
   <extends>QWidget</extends>
   <header>CircProgress</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ofen.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>backButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>724</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
